Cooking/index.html

```diff
@@ -8,119 +8,135 @@
 	</head>
 	<body>
 		<header>
```

Comment:
![image](https://user-images.githubusercontent.com/67651737/199071655-41b75766-e65d-48e0-a5ca-85d3f05f600f.png)
header must be full width